src/v2i-hub/PedestrianPlugin/manifest.json

```diff
@@ -32,6 +32,29 @@
 			"key":"WebServicePort",
 			"default":"9000",
 			"description":"Port at which Web service exists"
+		},
+		{
+			"key":"DataProvider",
+			"default":"FLIR",
+			"description":"Possible data providers we handle - FLIR or PSM XML data"
+		},	
+		{
+			"key":"WebSocketIP",
+			"default":"192.168.55.183",
+			"description":"IP address of the camera to connect to for data"
+
+		},
+		{
+			"key":"WebSocketURLExt",
```

Comment:
Small, non-critical note. I think for most configuration parameters for other connection plugins we use 'host' and 'port' as configuration parameter names so maybe we could do something similar here. Like WebSocketHost and WebSocketPort.

Reply:
Done